src/app/components/content-action-bar/ContentActionBar.test.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import ContentActionBar from "./ContentActionBar.jsx";
+import renderer from "react-test-renderer";
+import { mount } from "enzyme";
+
+describe("ContentActionBar", () => {
+    describe("is created stuck at bottom with a single action button, cancel and with unsaved changes", () => {
```

Comment:
describe usually should be used as a context description
like 
```suggestion
    describe("when props are passed ", () => {
```
I don't think it should be used in this condition. we have simple case here: only testing props being passed. I would remove it as it doesn't bring any value

Reply:
In regards to the testing comments, I don't think I quite follow, but have made some changes I hope are in line with your comments.

To me the description is to describe a unit of functionality and nested describes can explain a sub functionality of a testable unit. In this instance the unit of functionality was given a specific set of props does it function the way expected (the 'function' being does the component display the correct classes and elements). Then in the second test we are testing that given a change of different props it displays the expected elements and classes.